CHANGES.txt

```diff
@@ -24,6 +24,11 @@ Features
 * Pass ``post`` object and ``lang`` to post compilers (Issue #2531)
 * Pass ``url_type`` into template's context.
 * Make thumbnail naming configurable with ``IMAGE_THUMBNAIL_FORMAT``.
+* There is a new plugin category ``Taxonomy`` which allows to easily
+  create new taxonomies. Some of the existing taxonomies (authors,
+  archives, indexes, page index, sections) have been converted to
+  the new system. Tags will be converted to the new system as soon
```

Comment:
Ok, that does work now. There's only a little difference to the baseline: the feed title in the generated RSS feeds changed slightly. Everything else's identical.